packages/flutter/test/material/data_table_test.dart

```diff
@@ -1742,7 +1742,7 @@ void main() {
     expect(
       find.ancestor(of: find.byType(Table), matching: find.byType(Container)),
       paints..rect(
-        rect: const Rect.fromLTRB(0.0, 0.0, width, height),
+        rect: const Rect.fromLTRB(borderVertical / 2, borderHorizontal / 2, width - borderVertical / 2, height - borderHorizontal / 2),
```

Comment:
They are flipped because of this https://github.com/flutter/flutter/issues/122034#issuecomment-1456808780

I don't know which name would fit better. `borderVerticalLines`? Or I just flip them and `vertical: borderHorizontal`?

Reply:
They don't appear flipped to me, and make sense as-is.  They're referring to the border size on the vertical and horizontal sides of the rect, right?